### ISSUE TITLE
Tracking foreign data wrappers from another schema, v 1.2.1

### ISSUE THREAD

**Issue by rhideg** [hasura/graphql-engine#4714]
* We used the Hasura 1.2.1 version and all of a sudden the previously tracked tables from two schemas disappeared
* The tables in the public schema remained tracked, but we were not able to retrack the lost tables (they were still in the database)
* After reloading the metadata, resetting everything we could, deleting all data and starting from scratch, but nothing worked
* We resolved this issue by downgrading Hasura to v. 1.1.1, but with the new version it's not working like it used to

**Comment by rikinsk** [comment]
@rhideg Just to confirm, when you say the tables dissappeared, are they only not visible on the console or are they not being exposed over the GraphQL API as well?

**Comment by rhideg** [comment]
> @rhideg Just to confirm, when you say the tables dissappeared, are they only not visible on the console or are they not being exposed over the GraphQL API as well?
> 
> _edit:_ I guess the 2nd point indicates that they arent getting exposed over the API as well but was just making sure

So first we noticed some weird thing with the foreign data wrappers in the queries, I've made a relation on the portal to one of these tables from the public schema and the query was missing some data and on the fifth or sixth click all the required data was there. At this point I was curious what was happening and checked the track tables and the previously tracked tables from those two schemas were gone. I was able to query it with the graphiql but didn't see at the data menu. I tried reloading the metadata, then refreshing everything and retracking all tables, but I couldn't track them anymore. We struggled with it for a bit, then rolled back to v. 1.1.1, now everything is fine, but I don't want to miss out on new versions. :/
